src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -345,10 +350,74 @@ def test_0002_bz1209483(self, multihost, backupsssdconf):
         assert "foo2:*:2001:2001::/home/foo2:/bin/bash" \
                in execute_cmd(multihost, "getent passwd -s "
                                          "sss foo2").stdout_text
-        ssh1 = SSHClient(client_e, username="foo2", password="Secret123")
+        ssh1 = SSHClient(client_e, username="foo2",
+                         password="Secret123")
```

Comment:
There is a PR currently in review to remove use of paramiko, we can use `auth_from_client` here
https://github.com/SSSD/sssd/pull/6186/files#diff-e4ef93636bd6c6a155e832b3ce4a90402ccfc0f9d92e08448a2d7804f63d8593R403